### ISSUE TITLE
User is unable to access all the content available in 'React Native Gallery Preview' application using arrow keys in scan mode and caps +arrow key navigation.

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#710]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Narrator

Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery-Preview app.
Now try to access all the content available in 'React Native Gallery Preview' application using arrow keys in scan mode and caps +arrow key navigation.
Now observe the Narrator behavior
Actual Result:
User is unable to access all the content available in 'React Native Gallery Preview' application using arrow keys in scan mode and caps +arrow key navigation.

Note: 
While navigating in headings mode using 'H' & 'Shift+ H' keys, Narrator simply announcing as "unable to move narrator cursor"
While navigating in links mode using 'K' and 'Shift+K' keys, for first link focus is going and announcing the first link information and hitting k again, it is announcing as "unable to move narrator cursor"
User is able to navigate in 'Buttons' mode navigation.
Expected Result:
User should able to access all the content available in 'React Native Gallery Preview' application using arrow keys in scan mode and caps +arrow key navigation.

User Impact:
User who rely on Screen reader will face difficulties, If all the content available in 'React Native Gallery Preview' application is not accessible in scan modeand caps +arrow key navigation.

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
